Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6-alpine
+
+COPY requirements.txt /app/
+
+RUN true \
+	&& pip install gunicorn \
+	&& pip install -r /app/requirements.txt \
+	&& true
+
```

Comment:
We are not installing, the app directory is mounted to `/app`. Doing it this way removes necessity of installation and using `gunicorn` ensures that when the dev modifies app file on the host, `gunicorn` will reload the app.